07-github.md

```diff
@@ -180,6 +180,15 @@ If someone else had pushed some changes to the repository on GitHub,
 though,
 this command would download them to our local repository.
 
+> ## GitHub GUI {.challenge}
+> 
+> If you've been following along, browse to your planets repository on GitHub.
```

Comment:
Let's assume everyone is using the same repo. Could you change the first sentence to just:

"Browse to your `planets` repository on GitHub."

and remove the next sentence "Otherwise..."